docker.Makefile

```diff
@@ -128,22 +128,15 @@ build-e2e-image:
 	IMAGE_NAME=$(E2E_IMAGE_NAME) VERSION=$(VERSION) docker buildx bake e2e-image
 
 .PHONY: test-e2e
-test-e2e: test-e2e-non-experimental test-e2e-experimental test-e2e-connhelper-ssh ## run all e2e tests
+test-e2e: test-e2e-always-experimental test-e2e-connhelper-ssh ## run all e2e tests
```

Comment:
GitHub action names are so horrible 😞 - wondering if we could get rid of the double `e2e/e2e` somehow in that matrix 🤔 

<img width="626" alt="Screenshot 2025-03-10 at 11 44 34" src="https://github.com/user-attachments/assets/26570df2-279e-40ca-ad8a-b62b45eaaf08" />


That may require more changes though, but perhaps a shorter name for this one, e.g. `test-e2e-local` (because the other option in the matrix is `test-e2e-connhelper-ssh`)?

Reply:
changed it to `e2e/tests <name`